src/main/java/org/spongepowered/api/util/command/CommandSource.java

```diff
@@ -44,6 +46,17 @@
     String getName();
 
     /**
+     * Gets the locale used by this command source. If this
+     * {@link CommandSource} does have a {@link Locale} configured or does not
+     * support configuring a {@link Locale}, the system default is used.
+     *
+     * @return The locale used by this command source
+     */
+    default Locale getLocale() {
+        return Locale.getDefault();
```

Comment:
@ST-DDT 
Vanilla uses `en_US` as the default language throughout the client and server (and `en_US` is the fallback if the client does not have a desired language).

It's also worth nothing that [`Locale.US` is used as the default translation language](https://github.com/SpongePowered/SpongeAPI/blob/master/src/main/java/org/spongepowered/api/text/translation/Translation.java#L58).
A constant, or method, that returns `Locale.US` should be defined and used throughout, in my opinion. @Zidane